docs/tutorials/add-oauth2-provider.md

```diff
@@ -173,9 +169,34 @@ class [PROVIDER NAME] extends OAuth2
 
 Please mention in your documentation what resources or API docs you used to implement the provider's OAuth2 protocol.
 
-## 3. Test your provider
+## 3. Add provider to console
+
+### 3.1 Add provider logo
+
+To add the logo for the provider you are implementing, you will need to create a separate pull request in [`appwrite/console`](https://github.com/appwrite/console) repository.
```

Comment:
We might need to be more verbose here.

We should ask the reader to create a new branch on the console ( since we will be referring to it in the next step )

Reply:
Yeah let's make it s beginner friendly as possible 👍